pkg/app/piped/platformprovider/ecs/routing_traffic.go

```diff
@@ -14,9 +14,32 @@
 
 package ecs
 
+import (
+	"github.com/aws/aws-sdk-go-v2/service/elasticloadbalancingv2/types"
+)
+
 type RoutingTrafficConfig []targetGroupWeight
 
 type targetGroupWeight struct {
 	TargetGroupArn string
 	Weight         int
 }
+
+func (c RoutingTrafficConfig) hasSameTargets(forwardActionTargets []types.TargetGroupTuple) bool {
+	if len(c) != len(forwardActionTargets) {
+		return false
+	}
+
+	cMap := make(map[string]bool)
+	for _, item := range c {
+		cMap[item.TargetGroupArn] = true
+	}
+
+	for _, target := range forwardActionTargets {
+		if !cMap[*target.TargetGroupArn] {
+			return false
+		}
+	}
```

Comment:
[nit] It would be nice to use `struct{}` empty struct :)
This is more efficient because it does not allocate memory.

```suggestion
	cMap := make(map[string]struct{})
	for _, item := range c {
		cMap[item.TargetGroupArn] = struct{}{}
	}

	for _, target := range forwardActionTargets {
		if _, ok := cMap[*target.TargetGroupArn]; !ok {
			return false
		}
	}
```

Reply:
@ffjlabo Thank you so much! I fixed

Reply:
@ffjlabo 
I fixed again to use `tc:=tc` as you suggested instead of `tc:=testcase`
because I found an article that recommends to use the same name.

https://go.dev/wiki/CommonMistakes